api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -982,7 +982,25 @@ public DbUser updateRasLinkLoginGovStatus(String loginGovUserName) {
   @Override
   public DbUser confirmProfile() {
     final DbUser dbUser = userProvider.get();
-    dbUser.setProfileLastConfirmedTime(new Timestamp(clock.instant().toEpochMilli()));
-    return userDao.save(dbUser);
+    return updateUserWithRetries(
```

Comment:
Missed this in #4973 

Updated this to use `updateUserWithRetries()` which will cause an access tier/expiration recalculation.  Without doing this, the access tier recalculation will happen at some arbitrary future time (nightly at the latest) - now the update will have immediate effect.  FYI @freemabd @petesantos @dmohs